app/src/main/java/com/weatherxm/ui/common/UIModels.kt

```diff
@@ -684,6 +688,17 @@ enum class RewardTimelineType {
     END_OF_LIST
 }
 
+enum class PhotoSource {
+    CAMERA,
+    GALLERY;
+
+    val exifUserComment: String
+        get() = when (this) {
+            CAMERA -> "wxm-device-photo-camera"
+            GALLERY -> "wxm-device-photo-library"
+        }
+}
+
```

Comment:
This can be simplified to:
```
enum class PhotoSource(val exifUserComment: String) {
    CAMERA("wxm-device-photo-camera"),
    GALLERY("wxm-device-photo-library")
}
```